lib/views/help/privacy.es.html.erb

```diff
@@ -90,6 +92,15 @@
       <p>
         derechoapreguntar.org es un archivo público permanente de acceso a solicitudes de información realizadas a instituciones Nicaragüenses. Aunque la respuesta a tu consulta ya no te parezca de utilidad, puede resultar de interés para otras personas. Por este motivo no eliminamos las solicitudes, excepto en circunstancias excepcionales relacionadas con información privada.
       </p>
+      <p>
+          <strong>Nota:</strong> El usuario de esta Web en cualquier momento y por las razones que considere oportunas,  podrá solicitar al administrador de derechoapreguntar.org, retirar y/o eliminar de esta Web una o varias solicitudes de información hechas por el mismo usuario, ante cualquier entidad. Puede escribirnos a <a href="mailto:contacto@derechoapreguntar.org">contacto</a>.
+    </p>
+    <p>
+          Pese a este retiro de la Web o de las solicitudes hechas por el usuario, el administrador de derechoapreguntar.org no tiene facultades legales ni posibilidades técnicas de retirarla de la entidad que recibió la solicitud correspondiente, ni de detener el curso de la misma en la entidad respectiva. <strong>En estos casos derechoapreguntar.org no se hace responsable de monitorear el curso de la solicitud retirada y/o eliminada de la web a solicitud del usuario.</strong>  
+    </p>
+    <p>
+          El administrador de derechoapreguntar.org podrá retirar de la Web cualquier solicitud de información por razones de fuerza mayor. 
```

Comment:
We should probably link these back to the homepage. You can do that with `<a href="/">derechoapreguntar.org</a>`.